types/options.pp

```diff
@@ -1 +1,3 @@
+# @summary keepalived::options
```

Comment:
The type is not this but `Keepalived::Options` but explaining what is describing the data type is better.